Document CoC Member selection (#1373)
openjs-foundation/cross-project-council: conduct/COC_TEAM_CHARTER.md

The CPC is responsible for ensuring that the CoC Team is as representative and diverse as possible. To do so, the CPC commits to reach out proactively to individuals from underrepresented groups.

> Candidates can self-nominate to be selected as an alternate community member, a primary member, or both. If there is an objection to a nomination and consensus cannot be reached on that nomination then a vote will occur on that nomination. 

Reviewer: ```suggestion
Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.
```

> Once the nomination period has ended, primary and alternate members will be selected based on consensus (and voting if necessary). Our policy on consensus and voting is outlined in our [Charter](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making). 

Reviewer: ```suggestion
Once the nomination period has ended, primary and alternate members will be selected based on consensus. If there is an objection to a nomination and consensus cannot be reached then a vote will occur for that nomination. Our policy on consensus and voting is outlined in our [Charter](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making). 
```

Candidates can self-nominate to be selected as an alternate community member, a primary member, or both.

Once the nomination period has ended, primary and alternate members will be selected through [consensus][decision-making] during a private CPC meeting. If there is an objection to a candidate being selected and consensus cannot be reached, a [vote][voting] will occur for that candidate.

> [voting]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-10-voting

Author: ```suggestion
[decision-making]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making
[voting]: https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-10-voting
```
